docs/chain/collator.md

```diff
@@ -323,176 +297,175 @@ Secret phrase `very secure private key you should not use the example private ke
-2. Submit the following extrinsic: parachainStaking -> joinCandidates(stake)
-3. Insert the staked KILT amount for the collator 
+1. Select the collator's KILT address as the extrinsic submitter (the *using the selected account* field)
+2. Submit the following extrinsic: `parachainStaking -> joinCandidates(stake)`
+3. Insert the staked KILT amount for the collator
+
+:::info
+
+A recent change in the blockchain metadata resulted in a change in the UI regarding how balances are shown. In the current version of PolkadotJS Apps, specifying 1 KILT requires adding 15 trailing `0`s. So, for instance, 1 KILT needs to be written as `1000000000000000`, while 10,000 KILT would be written as `10000000000000000000`.
+
+:::
 
 ![](/img/chain/parachainStaking-joinCandidates.png)
 
-A collator can check the current `SelectedCandidates` to see the position and required staked amount to join the network. 
+A collator can check the current top candidates to see the position and required staked amount to join the network. 
 
 `Developer -> Chain state -> Storage`
 
-1. Selected state query: `parachainStaking -> selectedCandidates(): Vec<AccountId>`
-2. Add the item with the "+" on the right side
+1. Selected state query: `parachainStaking -> topCandidates(): ParachainStakingSetOrderedSet`
+2. Execute the query by pressing the "+" button on the right side
 
-If the collator has enough self-stake and delegator stake they can be selected to collate.
-Once the collator has been chosen, the collator will be added to the `SelectedCandidate` pool.
+If the collator has enough self-stake and delegator stake it will be selected to collate.
+Once the collator has been chosen, it will be part of the `topCandidates()` set.
 A time period of two sessions must pass before the collator will be authoring blocks, e.g.
 after the rest of current session and the entire next one.
 
 ![](/img/chain/session-validators.png)
 
-### Stake on collator
+### Change a Collator's Stake
 
-A collator can increase/decrease their stake.
-The corresponding methods can be found as an extrinisc under `parachainStaking > candidateStakeMore / candidateStakeLess`.
+A collator can increase/decrease their stake, always within the limits of the minimum and maximum allowed stake amounts.
+The corresponding extrinsics for these operations are `parachainStaking -> candidateStakeMore(more)` and `parachainStaking -> candidateStakeLess(less)`.
 
 ![](/img/chain/parachainStaking.png)
 
 ## Monitoring
 
-It would be ideal if monitoring is done from a different host.
-However in cases of resource constraints it would be better to run these monitoring tools on the same machine where your collator is running than not having.
-You can either run grafana by your own or subscribe free [grafana cloud ](https://grafana.com/products/cloud/) option. 
+It would be ideal if the host being monitored is not the host monitoring, a.k.a., if the monitoring process does not run on the same host as the collator process.
```

Comment:
I think the comma is after i.e. [Had to duckduckgo that](https://www.wikihow.com/Use-i.e.-in-a-Sentence)

```suggestion
It would be ideal if the host being monitored is not the host monitoring i.e., the monitoring process does not run on the same host as the collator process.
```

Reply:
Yeah wanted to be a bit more informal here, but ok 🧐 It goes both before and after tho.